chat-plugins/economy.js

```diff
@@ -64,98 +64,98 @@ exports.commands = {
 		if (userid.length > 19) return this.sendReply("/wallet - [user] can't be longer than 19 characters.");
 
 		Economy.readMoney(userid, money => {
-			this.sendReplyBox(SG.nameColor(target, true) + " has " + money + ((money === 1) ? " " + currencyName + "." : " " + currenyPlural + "."));
+			this.sendReplyBox(SG.nameColor(target, true) + " has " + money + ((money === 1) ? " " + currencyName + "." : " " + currencyPlural + "."));
 			//if (this.broadcasting) room.update();
 		});
 	},
 
-	gc: 'givecurrency',
-	givecurrency: function (target, room, user) {
```

Comment:
Other servers use SGs repo too, so we cant just refactor this over to stardust. How about setting givestardust, gs, takestardust, ts, ect as aliases of givecurrency ect?